datasetinsights/evaluation_metrics/metrics_utils.py

```diff
@@ -0,0 +1,47 @@
+import collections
+
+import numpy as np
+
+
+def mean_metrics_over_iou(metric_per_iou):
+    """Calculate mean value over IOUs.
+
+    Args:
+        metric_per_iou (dict): metric records for each iou
+
+    Returns (float):
+        mean metric values over IOUs
+    """
+    mean_sum = 0
+    for mean_ap in metric_per_iou:
+        result = mean_ap.compute()
```

Comment:
Why is `metric.compute` called here? This is not the responsibility of the `mean` operation, is it?

Reply:
Will move `compute()` out of this method.